### PR TITLE
Add tests for verifying file permissions and hashes with RPM

### PR DESCRIPTION
**Pull request by mildass**
#### Description:
Add two tests, that should pass on fresh system, and one script, after which should test fail.

Tests for:
- Verify and Correct File Permissions with RPM rule
- Verify File hashes with RPM rule

Script for:
- Verify and Correct File Permissions with RPM rule

### CODE REVIEW

**Comment by jan-cerny** [comment]
@openscap-ci add to whitelist

**Comment by mpreisler** [comment]
NACK. Fix your git author in the commit:
```
From: Milan Lysonek <mlysonek@localhost.localdomain>
```

File: tests/data/group_system/group_software/group_integrity/group_software-integrity/group_rpm_verification/rule_rpm_verify_hashes/fresh_system.pass.sh [hunk 1]
```diff
@@ -2,6 +2,11 @@
 #
 # profiles = xccdf_org.ssgproject.content_profile_pci-dss
 
-# we expect the system is in good shape
-# TODO: add check here, so this script fails when not true
-true
+# verify all packages from rpm 
+result=$(rpm -Va | grep '^..5')
+
+if [ -z "$result" ]; then
```
Comment (mpreisler) [inline tests/data/group_system/group_software/group_integrity/group_software-integrity/group_rpm_verification/rule_rpm_verify_hashes/fresh_system.pass.sh]: this is a bash anti-pattern. you are running `[` which returns 0 exit code or 1, 0 for true. then you check that it's 0 (true) and then you run true wihch again exists with 0. you can simplify this to just running `[` and that's all.

**Comment by dahaic** [comment]
Agree, tester should prepare the machine to pass. It's more efficient that way.

**Pull request merged**